src/main/webRequest/webRequestHandler.ts

```diff
@@ -6,28 +6,31 @@ import {EventEmitter} from 'events';
 import log from 'electron-log';
 
 export class WebRequestHandler<T, T2> extends EventEmitter {
-    modifyCallbackObject: (details: T, callbackObject: T2, result: T2) => T2;
+    protected modifyCallbackObject: (details: T, callbackObject: T2, result: T2) => T2;
```

Comment:
I went with `protected` in case people wanted to inherit this class. Doubt it'll make a difference long term, but I was trying to be as correct as I could with accessor modifiers :)